example/AR_tag_aruco.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import cv2
+import numpy as np
+from cv2 import aruco
+import core
+
+# define an empty custom dictionary with
+aruco_dict = aruco.custom_dictionary(2, 5, 1)
+# add empty bytesList array to fill with 3 markers later
+aruco_dict.bytesList = np.empty(shape=(2, 4, 4), dtype=np.uint8)
+
+# add custom markers as defined by the ALVAR library
+mybits = np.array(
+    [
+        [1, 1, 0, 1, 1],
+        [1, 1, 0, 1, 1],
+        [1, 0, 1, 0, 1],
+        [1, 1, 1, 1, 1],
+        [
+            1,
+            1,
+            1,
+            1,
+            1,
+        ],
+    ],
+    dtype=np.uint8,
+)
+aruco_dict.bytesList[0] = aruco.Dictionary_getByteListFromBits(mybits)
+
+mybits_1 = np.array(
+    [
+        [1, 1, 0, 1, 1],
+        [1, 1, 0, 1, 1],
+        [1, 0, 1, 0, 1],
+        [0, 0, 1, 1, 0],
+        [
+            1,
+            1,
+            1,
+            0,
+            1,
+        ],
+    ],
+    dtype=np.uint8,
+)
+aruco_dict.bytesList[1] = aruco.Dictionary_getByteListFromBits(mybits_1)
+
+
+def main() -> None:
+    """
+    Main function for video stream script, tests streaming/recording zed footage
+    """
```

Comment:
In general, all examples probably need more thorough documentation, at least in the main function docstring. This and the following 3 examples all have practically the same docstring and it is hard to tell what each is supposed to do.